openapi/elasticsearch/Module.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+This is a generated connector for [Elasticsearch](https://cloud.elastic.co/api/v1/api-docs/spec.json) OpenAPI specification. 
+
+Elasticsearch API allows you to store, search, and analyze data with ease at scale.
+
+## Prerequisites
+
+Before using this connector in your Ballerina application, complete the following:
+
+* Create a [Elasticsearch Account](https://cloud.elastic.co/registration).
+* Obtain tokens by following [this guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api.html#security-token-apis).
+
```

Comment:
Can we add a sample use-case since this connector has tested ?

Reply:
Added sample to `quickstart`